db/migrate/20231120164346_add_unique_index_on_member_uid_in_members.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexOnMemberUidInMembers < ActiveRecord::Migration[6.1]
+  def change
+    add_index :members, [:organization_id, :member_uid], unique: true
```

Comment:
@nflorentin why is necessary this index?

Reply:
more about it in @sauloperez comment : https://github.com/coopdevs/timeoverflow/issues/311#issue-292798958

Reply:
ah right!